prow/tide/tide.go

```diff
@@ -727,7 +731,7 @@ func accumulate(presubmits map[int]sets.String, prs []PullRequest, pjs []kube.Pr
 				overallState = pendingState
 			}
 		}
-		if overallState == successState {
+		if overallState == successState || headContextOnly {
```

Comment:
If there are no presubmit ProwJobs to list above, won't we always be in `successState`?